scss/bootstrap/bootstrap.scss

```diff
@@ -20,4 +20,5 @@
 #{$outside-namespace-marker} {
   @import 'components/outside-namespace/*';
   @import 'overrides/crm/outside-namespace/*';
+  @import 'SCSSROOT/org.civicrm.shoreditch/scss/angular/overrides/outside-namespace/ALL';
```

Comment:
why do we need SCSSROOT at all here? you could use the relative path, couldn't you

Reply:
@AkA84 it does not work for some reason. We don't have time right now to discover why.